docs-src/go/generated/add-sleep-for-one-minute.md

```diff
@@ -0,0 +1,68 @@
+---
+id: add-sleep-for-one-minute
+title: Add a call to sleep
+sidebar_label: Add sleep call
+description: Add a call to sleep for one minute to the beginning of the Workflow.
+tags:
+- timer
+- sleep
+- logger
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-go/blob/port_replay_test_dacx/backgroundcheck_replay/workflows/backgroundcheck.go. -->
+
+In the following sample, we add a couple of logging statements and a Timer to the Workflow code to see how this affects the Event History.
```

Comment:
Is Timer a proper noun at the time you read this document?
If I was reading this for the first time, `Timer` isn't anything special and in the Go SDK it is a sleep, so the reader won't ever make the connection about the proper noun.

Reply:
Could link to Timer docs here for that matter

Reply:
We don't have anything to link to - 
BUT, we probably should because a Temporal Timer is one that is durable. Going to keep it proper and plan to come back and update when we have a feature reference.